### DISCUSSION TITLE
Не видит участки

### DISCUSSION THREAD

**Discussion by Greay7891** [Chia-Network/chia-blockchain#7009] [Support]
Добрый день. После перезагрузки компьютера, программа "chia blockchain" не распознаёт созданные участки.

**Comment by vgalbur** [comment]
У меня тоже самое... В чём может быть проблемма?

**Comment by loppefaaret** [comment]
Could either of you provide a screenshot of the problem ?

**Reply by Greay7891** [reply]
![Снимок экрана (1)](https://user-images.githubusercontent.com/86594412/123909038-1e762580-d981-11eb-990b-28163f51f085.png)
![Снимок экрана (2)](https://user-images.githubusercontent.com/86594412/123909042-1f0ebc00-d981-11eb-8f74-4d9b4d1ad5c8.png)

**Reply by Greay7891** [reply]
После того как "chia blockchein" перестал видеть участки, я создал новый плот. Но программа его тоже не распознаёт.

**Comment by loppefaaret** [comment]
Try and reset the configuration:
WINDOWS COMPLETE CONFIG RESET

Close down the Chia app, and open up a PowerShell window, and execute the below commands one at a time:
```
mv ~\.chia\mainnet ~\.chia\old

cd ~\AppData\Local\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon\

.\chia.exe init

mv ~\.chia\old\db ~\.chia\mainnet\

mv ~\.chia\old\wallet ~\.chia\mainnet\
```

**Reply by Greay7891** [reply]
Спасибо. Это помогло.

**Reply by Maxim-paa** [reply]
не получается показывает отказ в доступе строка 1

**Reply by Maxim-paa** [reply]
PS C:\Users\Сука> mv ~\.chia\mainnet ~\.chia\old
mv : Отказано в доступе по пути "C:\Users\Сука\.chia\mainnet".
строка:1 знак:1
+ mv ~\.chia\mainnet ~\.chia\old
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\Users\Сука\.chia\mainnet:DirectoryInfo) [Move-Item], IOException
    + FullyQualifiedErrorId : MoveDirectoryItemIOError,Microsoft.PowerShell.Commands.MoveItemCommand

**Reply by Maxim-paa** [reply]
все я разобрался огромное спасибо!

**Reply by AlonsoKostia** [reply]
скажи пожалуйста, как разобрался? всю голову сломал, что не так..

**Comment by vgalbur** [comment]
Привет! Помогло только рестарт всего кошелька... Пришлось по новой ждать
полной синхронизации...

mv ~\.chia\mainnet ~\.chia\old

cd ~\AppData\Local\chia-blockchain\app-1.1.7\resources\app.asar.unpacked\daemon\

.\chia.exe init

mv ~\.chia\old\db ~\.chia\mainnet\

mv ~\.chia\old\wallet ~\.chia\mainnet\


вт, 27 июл. 2021 г. в 17:35, AlonsoKostia ***@***.***>:
